src/main/java/com/ernie/TicketApp/service/TicketService.java

```diff
@@ -23,32 +26,88 @@ public static void main(String[] args) {
 
-        Client client = new Client();
-        client.printRole();
-        client.print();
-        client.getTicket(ticketLimited);
+        System.out.println("============================= HW 6 (Data Structures) ====================================");
+        System.out.println("Testing custom ArrayList");
+        MyArrayList myArrayList = new MyArrayList();
+        myArrayList.put("Element 1");
+        myArrayList.put("Element 2");
+        myArrayList.put("Element 3");
+        myArrayList.put("Element 4");
+        myArrayList.put("Element 5");
+        myArrayList.put("Element 6");
+        myArrayList.put("Element 7");
+        myArrayList.put("Element 8");
+        myArrayList.put("Element 9");
+        myArrayList.put("Element 10");
+        myArrayList.put("Element 11");
 
-        Admin admin = new Admin();
-        admin.checkTicket(ticketFull);
+        myArrayList.printAll();
+        System.out.println(myArrayList.size() + " " + myArrayList.capacity() + "\n");
+        System.out.println(myArrayList.getByIndex(4) + "\n");
 
-        ticketEmpty.setId(null);
-        System.out.println(ticketEmpty.getId());
-        NullCheckUtility.validateNotNull(ticketEmpty);
+        myArrayList.deleteByIndex(4);
+        myArrayList.printAll();
+        System.out.println(myArrayList.size() + " " + myArrayList.capacity() + "\n");
+
+        myArrayList.print();
+
+        System.out.println("Testing custom HashSet");
+        MyHashSet myHashSet = new MyHashSet();
+        myHashSet.put("Element 1");
+        myHashSet.put("Element 2");
+        myHashSet.put("Element 3");
+        myHashSet.put("Element 3");
+        myHashSet.put("Element 3");
+
+        myHashSet.print();
+
+        System.out.println(myHashSet.contains("Element 4"));
+        System.out.println(myHashSet.contains("Element 2"));
+
+        myHashSet.delete("Element 1");
+
+        myHashSet.print();
+
+        ticketFull.setTicketType(TicketType.DAY);
+
+        System.out.println("============================= HW 8 (JDBC) ====================================");
+
+        UserDAO userDAO = new UserDAO();
+        TicketDAO ticketDAO = new TicketDAO();
+
+        userDAO.saveUser(client1);
+
+        userDAO.deleteUserById(UUID.fromString("d967d60c-167b-43b0-ba13-d08f0c59a721"));
+        userDAO.saveUser(admin);
+
+        User userFromDB = userDAO.fetchById(UUID.fromString("4bf70e48-683d-45a8-8427-f4613f55ade3"));
+        System.out.println(userFromDB);
+
+        ticketDAO.saveTicket(ticketFull, userFromDB);
+
+        Ticket ticketFromDB = ticketDAO.fetchById(UUID.fromString("278e7e12-e84e-4c95-8217-46d7a0dc5808"));
+        ticketFromDB.print();
+        ticketDAO.updateTicketType(ticketFromDB, TicketType.WEEK);
+        Ticket updatedTicketFromDB = ticketDAO.fetchById(ticketFromDB.getId());
+        updatedTicketFromDB.print();
+
+        Ticket ticket1 = new Ticket();
+        ticket1.setId(UUID.fromString("34227935-f280-41e9-bba9-bbcdc69f9dba"));
```

Comment:
do not use hardcoded UUIDs. instead of that, retrieve them from db. for example, you can add method to retrieve all values and use one of them:
1) in UserDAO:
```java
    public List<User> getAll() {
        List<User> users = new ArrayList<>();
        String SQLStatement = "SELECT * FROM user_info";
        try (PreparedStatement preparedStatement = ConnectionConfig.getConnection().prepareStatement(SQLStatement)) {

            ResultSet resultSet = preparedStatement.executeQuery();
            while (resultSet.next()) {
                String name = resultSet.getString("name");
                User user = new User(name);
                user.setId(UUID.fromString(resultSet.getString("id")));
                user.setCreationDateTime(LocalDateTime.parse(resultSet.getString("creation_date")));
                users.add(user);
            }
        } catch (SQLException e) {
            e.printStackTrace();
        }
        return users;
    }
```

2. In TicketService:
instead of
```java
 userDAO.deleteUserById(UUID.fromString("d967d60c-167b-43b0-ba13-d08f0c59a721"));
```

use
```java
List<User> users = userDAO.getAll();
if(users.size() > 0) {
  userDAO.deleteUserById(users.get(0).getId());
}
```

Reply:
fixed in https://github.com/my20yeezy/Learning-at-Andersen/pull/4/commits/1f9c7997a17b1927a39fb635693f3f25e816f038